### DISCUSSION TITLE
My dump of stuff for lazer

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#28930] [Ideas]
Im going to go straight to the point

**Beatmap editor**

Continued in here: https://github.com/ppy/osu/discussions/29014

**'Now playing'** - aka F6.

1. **Small bug i found** - for some reason, when trying to skip 'new beginings' (aka the tutorial), the game doesn't allow you to do so.
Its not the last alphabetically, so its definitely not the last song - in fact, it displays as the first on my list, so that makes even less sense. **Its more of a bug, possibly even just a problem with my game, but im already dumping everything here, so ill post it on the forum for issues/help later**.
2. **Sort** - Add a feature to it that sorts the played songs in a specific way. They can be: Alphabeticall, Date added, Lenght, genre, etc.
3. **Add a speed modifier to it**, with a pitch changer to turn off and on. This can look like im making osu into a small music player, but i think most of the people use it like that too - Im literally in the osu menu with the songs automatically changing and just playing in the background. Adding a Speed modifier would be able to enchance this small feature osu. also, while your add it, add a nightcore and daycore 'mod' to it - since they change the speed, but also add some other stuff to the music (atleast NC does, im not sure about DC).
4. Speaking of making osu into a mini music player, please add a 'repeat song' feature to this.

**Rulesets**

1. **Small adjustments to the ruleset select sfx** - I like the new ruleset sound seletc sound, ill admit, but why is osu!standard so loud? I love the taiko and catch sfx for select, but mania and standard are killing my ears (highlighting standard) Mania is fine, but it would sound better (imo) if we moved the sfx thats on all rulesets to the last note, for that epic feel that taiko gives. For standard - make its sfx quieter, or also move it a bit so the main sfx is after the 'finish hs', so it won't be killing everyones ears.
2. **Highlight to point 2 of the 'beatmap editor' section**.

**Star rating**

1. **Make AR, OD (Accuracy now), and HD count towards the increase/decrease of star rating**. This is an issue to me especially when using Difficulty Adjust, because only Circle size changes the star rating?? Why?
2. Sometimes star rating can feel inacurate for whatever reason, thats why addding a **community** sr would be good. the community could vote for the rating of the map. it could indicate which maps are harder than others with similar sr. Of course tho only a difference in 1* could be voted, so that people won't spam 10* on normal diffs, and other weird stuff. The vote wouldn't have a format of 'pick from 3 to 5'  on a map with 4*, you could pick values like 4,37* , 3,86* ect.

**Alt + f4**

1. Make it an option in the settings for the game to close immediately, or give you the small 'cutscene' it does when you close, without the 'do you want to close the game?' question. I know osu is addictive, but when i want to leave, i leave, sorry. Could be in the general settings tab. 

**Beatmap Votes**

1. Make a vote if the map/difficulty is good or bad based on community opinion. You'd have 5 choices: Very bad, Bad, Neutral, Good, Very good. The catch is, you have to get atleast an 'A' to be able to submit your vote, so people who just can't play the map can't say the map is Very bad, just because they can't play it.
2. Even if you don't stick to point 1 for every map, it would be cool to see it in the Daily challenge.

**Mods**

1. **catch** - Make floating fruits rankable, it changes only the fact that instead if going down, fruits go up. nothing else is changed, so why isn't it ranked? Why is mirror unranked?
2. **mania** - Using flash light (on argon) makes it imposible to see what rating you hit has, make it visible. Cover is the same thing as fade in, just even harder and customisable. In the mod select screen, the stats in the right down corrner display the CS stat, it should display the KC stat. Why is DS unranked?
3. **taiko** - Why is SW unranked? it doesn't make the gameplay easier or harder. also make more mods for it please,
4. **Standard** - put Tracable in the Difficulty Increase section, and give it a 0,06 modifier (the same as hidden) Make the BU mod somewhat transparent, so itll be atleast somewhat playable.
5. **All gamemodes** - make difficulty adjust make sense.

**Main menu**

1. When holding shift/ctrl (i don't remeber which key it was), in the timing menu in stable, when you held one of these keys it also made a sound on 1/2 of a beat (red lines). it would be a nice easter egg to add this to lazers main menu, that when you hold one of those keys and your cursor is on the osu! cookie, you get the same thing
2. This is my nitpicking, but when you select play in the main menu, the back button is a blue tone, similar to other buttons. sadly, this is also the case for when you pick edit, the button is also blue. This is me looking for smallest details, so i understand if this gets replied to with a big 'NO', but it would be a nice detail imo

I think this is it for now, if i find more things, ill edit them into the post.
Thanks for reading

Edit: Droped the chat part of the post,  as i didn't know about one of the features that were already in-game. Also added a 6th point to the beatmap editor part

**Comment by SupDos** [comment]
> Chat
> and make it take up the whole screen

You can already do this:

https://github.com/user-attachments/assets/8369bffe-365a-4ed1-89b4-630f0f4feb12

**Reply by Tenexxt** [reply]
Either that was added today or im just blind.
Either way, thanks!

**Comment by cl8n** [comment]
you might have to split these suggestions up a bit or else most people aren't going to read them.

- star rating 1: drawing the line between what should count for SR and what should count for pp is going to be arbitrary on some level no matter how you do it. the real answer is probably "that's just how it's been and everyone's used to it", but not without its merits -- for example OD not being included lets the player judge for themselves how they feel the increased acc difficulty changes the map for them, and another example, as an HD player I appreciate that this mod doesn't change SR because I know how that affects my play already. on one side you have the simplicity of bundling everything into one difficulty number, and on the other you have the ability to interpret a more personalized difficulty using many difficulty stats.
- star rating 2: I think this is too complicated for what is a not very useful result. for small deviations in perceived SR I think the community at large isn't qualified to tweak those numbers, and for large deviations that's just what it means for a map to be under/overweighted which should be addressed in the real SR system instead
- beatmap votes: the rating system needs a rework for sure but I think that's already covered in other issues
- mods 2 > "why is DS unranked": it seems like there is some agreement that DS should just be deleted in favor of having normal key count mods (for various reasons I won't elaborate on here)
- mods 5: not sure what you mean by that

the others i am indifferent or just have nothing to add

**Reply by Tenexxt** [reply]
At the time of this post i didn't really understand SR and how it works, so yea, i said some non-logicall stuff about it.

> mods 5: not sure what you mean by that

I meant like scoremultiplier, but i saw some issues already made for that so not the smartest take from me again

>beatmap votes: the rating system needs a rework for sure but I think that's already covered in other issues

This wasn't meant for the ranking system, but rather the overall info that the player can see about how others enjoy the map, tho now i see that this system would have faults. Ill word it better and adjust it so it would have more sense. also some sort of voting system is planed actually, because it was said that on the rework of tags some od them will be changed by the community, and not the mapper.

I probably need to get myself to think... and write properly.

**Reply by cl8n** [reply]
"rating system", not "ranking system" :^)  you can find some topics about that here: https://github.com/ppy/osu-web/issues?q=is%3Aissue+is%3Aopen+user+rating
